src/pages/VideoDetails/VideoDetails.page.jsx

```diff
@@ -0,0 +1,43 @@
+import VideoCard from 'components/VideoCard';
+import { useSingleYoutubeVideo } from 'hooks/youtube/video/useSingleYoutubeVideo';
+import React, { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import {
+  RelatedVideos,
+  VideoContainer,
+  VideoContent,
+  VideoPlayer,
+} from './VideoDetailsElements.styled';
+
+function VideoDetails() {
+  const { id } = useParams();
+  const { relatedVideos, isLoading, getRelatedVideos } = useSingleYoutubeVideo(id);
```

Comment:
Video suggestions are the same no matter which video we pick on the suggestion, my first thought was that maybe this id wasn't updating, but I also saw that if I pick a video from home or search fetches the same suggestions, so my second thought is about how are suggestions queried with gapi